docs/detections/use-osquery.asciidoc

```diff
@@ -0,0 +1,10 @@
+[[use-osquery]]
+= Use Osquery
```

Comment:
How about just "Osquery" for the section title, since this is a top-level landing page, not a procedure topic. Most of our other main section titles are just category names, not active verb statements.
```suggestion
= Osquery
```

Reply:
@jmikell821 I'm going to commit this for now, but lemme know if you think an alternative title should be used.